bin/describe/package_entries.ml

```diff
@@ -0,0 +1,24 @@
+open Import
+open Stdune
```

Comment:
This is already included in Import

Reply:
But almost all the files in `bin/describe` include both.

Reply:
Yes, it was recently added to Import I should go and clean those up.